berth_reservations/logging.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import time
+
+logger = logging.getLogger("requests")
+
+
+__all__ = ["LogRequestFilter", "RequestLogger"]
+
+
+class LogRequestFilter(logging.Filter):
+    """Since the requests are being logged by the RequestLogger,
+    we filter the django.server logs to avoid having duplicate logs"""
+
+    def filter(self, record: logging.LogRecord):
+        return record.name != "django.server"
+
+
+class RequestLogger:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        exec_time = time.time()
+        response = self.get_response(request)
+        exec_time = int((time.time() - exec_time) * 1000)
+
+        path = request.get_full_path()
+        status = response.status_code
+        method = request.method
+
+        message = f"{method} {path} {status}"
```

Comment:
Should the message be more descriptive, e.g. something like `f"Logged request: {method} {path} {status}"`?

Reply:
I think with all the fields it will be pretty clear that the log comes from a request.